doc/gnucobol.texi

```diff
@@ -414,26 +414,55 @@ Place the output into @var{file}.
 @node Source format
 @subsection Source format
 
-GnuCOBOL supports both fixed and free source format.
+GnuCOBOL supports fixed, free, variable, terminal, and COBOLX source formats.
 The default format is the fixed format.
 This can be overridden either by the
-@code{>>SOURCE [FORMAT] [IS] @{FIXED|FREE@}} directive,
+@code{>>SOURCE [FORMAT] [IS] @{FIXED|FREE|VARIABLE|XCARD|CRT|TERMINAL|COBOLX@}} directive,
 or by one of the following options:
 
 @table @code
-@item -free, -F
+@item -free, -F, -fformat=free
 Free format.  The program-text area starts in column 1 and
 continues till the end of line (effectively 255 characters
 in GnuCOBOL).
 
-@item -fixed
+@item -fixed, -fformat=fixed
 Fixed format.  Source code is divided into: columns 1-6, the sequence
 number area; column 7, the indicator area; columns 8-72, the program-text
 area; and columns 72-80 as the reference area.@footnote{Historically, fixed
 format was based on 80-character punch cards.}
 
+@item -fformat=variable
+Microfocus' Variable format.  Identical to the fixed format above except
+for the program-text area which extends up to column 250 instead of 72.
+
+@item -fformat=xcard
+ICOBOL xCard format.  Variable format with right margin set at column
+255 instead of 250.
+
+@item -fformat=crt
+ICOBOL Free-form format (CRT).  The program-text area may start in
```

Comment:
Isn't this the  X/Open free-form format (FR 130)? If it is we may move this up one entry, reference X/Open first, then ICOBOL.

Note: you can get the definition without cost (just registration needed) from https://publications.opengroup.org/c192

Reply:
X/Open required a slightly different debug indicator, so I've added a test for this format as well.
I changed the order of the formats in the documentation, but I'm not sure what's the main driver behind a specific choice; apart from fixed and free that come first, I was using a very loose and subjective measure of readability/usability of tghe sources ;-)

Reply:
BTW thanks for the link.